tests/test_functions.py

```diff
@@ -1883,6 +1875,7 @@ def historical_samples_example(self):
             samples=samples, mutation_rate=1, length=1e2, random_seed=12
         )
 
+    @pytest.mark.skip("Now that we don't allow unary nodes, this example no longer works")
```

Comment:
I don't understand why this is a problem with unary nodes? I think we just need to simplify the inferred_ts, don't we?